pombola/core/views.py

```diff
@@ -482,6 +482,7 @@ def position(request, pt_slug, ok_slug=None, o_slug=None):
 
     context = {
         'object':     title,
+        'ok_slug':    ok_slug,
```

Comment:
I'm not sure this is really worth changing for this pull request, but it might be more helpful for developers in the future if you passed the `OrganisationKind` object, rather than just the slug? (Also, if you called it `organisation_kind`, it'll be easier for people working in the templates to figure out what it's for.)